frontend/device/android/device.lua

```diff
@@ -83,7 +83,7 @@ local Device = Generic:new{
     home_dir = android.getExternalStoragePath(),
     display_dpi = android.lib.AConfiguration_getDensity(android.app.config),
     isHapticFeedbackEnabled = yes,
-    isDefaultFullscreen = function() return android.app.activity.sdkVersion >= 19 end,
```

Comment:
the intent of the function didn't change. The app starts in fullscreen on api19+. On previous api levels we need to do something to enforce fullscreen.